city_scrapers/spiders/cook_hospitals.py

```diff
@@ -21,118 +24,80 @@ def parse(self, response):
+                    'location': self._parse_location(subitem),
+                    'documents': self._parse_documents(subitem)
                 }
                 new_item.update(data)
-                new_item['status'] = self._parse_status(subitem, new_item['start_time'])
-                new_item['id'] = self._generate_id(data)
+                new_item['status'] = self._generate_status(new_item, '')
+                new_item['id'] = self._generate_id(new_item)
+                new_item['classification'] = self._parse_classification(new_item['name'])
                 yield new_item
 
-    def _parse_classification(self, item):
-        """
-        @TODO Not implemented
-        """
-        return 'Not classified'
-
-    def _parse_status(self, subitem, start_time):
-        """
-        @TODO
-        passed = meeting already started
-        tentative = no agenda posted
-        confirmed = agenda posted
-        Is this the right way to determine tentative vs confirmed?
-        Looks like agendas are usually posted a week before the meeting
-        """
-        if datetime.now().isoformat() > start_time.isoformat():
-            return 'passed'
-        agenda = subitem.xpath("following-sibling::td/a/@href").extract_first()
-        if agenda is None:
-            return 'tentative'
-        else:
-            return 'confirmed'
-
-    def _parse_location(self, subitem):
+    @staticmethod
+    def _parse_classification(name):
+        if 'BOARD' in name.upper():
+            return 'Board'
+        return 'Committee'
+
+    @staticmethod
+    def _parse_documents(subitem):
+        anchors = subitem.xpath("following-sibling::td//a")
+        documents = []
+        if anchors:
+            for a in anchors:
+                url = a.xpath('@href').extract_first()
+                notes = a.xpath('text()').extract_first()
+                notes = notes.lower() if notes is not None else ''
+                documents.append({'url': url, 'notes': notes})
+        return documents
+
+    @staticmethod
+    def _parse_location(subitem):
         """
         Parse location
         """
+        address = subitem.xpath('text()').extract()[1]
         return {
-            'url': '',
+            'address': address.strip(),
             'name': '',
-            'address': subitem.xpath('text()').extract()[1].strip(),
-            'coordinates': {'longitude': '', 'latitude': ''},
+            'neighborhood': '',
         }
 
-    def _parse_all_day(self, item):
-        """
-        It appears all events have a start and end time on the CCHHS website,
-        so this `all_day` is always false.
-        """
-        return False
-
-    def _parse_name(self, item):
+    @staticmethod
+    def _parse_name(item):
         """
         Get event name from item's text
         """
         return item.xpath('text()').extract_first().strip()
 
-    def _parse_description(self, subitem):
+    @staticmethod
+    def _parse_description(subitem):
```

Comment:
The signup spreadsheet said to remove the static description, but I didn't immediately see what should replace it on http://www.cookcountyhhs.org/about-cchhs/governance/board-committee-meetings/

Reply:
Looks like there's not much, so you could just return an empty string for the event description. We want to remove the static descriptions because they are actually descriptions of the host agencies and not of the events.